lntest/itest/lnd_test.go

```diff
@@ -2386,6 +2401,48 @@ func testUpdateChannelPolicy(net *lntest.NetworkHarness, t *harnessTest) {
 		)
 	}
 
+	// Now, to test that Carol is properly rate limiting incoming updates,
+	// we'll send one more update from Alice. Carol should reject this
+	// update as it only allows a single update per day, and a day has yet
+	// to elapse from the previous update.
+	prevAlicePolicy := *expectedPolicy
+	baseFee *= 2
+	expectedPolicy.FeeBaseMsat = baseFee
+	req.BaseFeeMsat = baseFee
+
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
+	defer cancel()
+	_, err = net.Alice.UpdateChannelPolicy(ctxt, req)
+	if err != nil {
+		t.Fatalf("unable to update alice's channel policy: %v", err)
+	}
+
+	// Wait for all nodes except Carol to have seen the policy updates for
+	// both of Alice's channels.
+	for i, graphSub := range graphSubs {
+		var expUpdates []expectedChanUpdate
+		if i != len(graphSubs)-1 {
```

Comment:
Could add a comment here that carol is added to `graphSubs` last which is why we check the final index. Alternatively, we could make it into a map for indexing, though that woudl expand the size of the diff a bit.